packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -47,7 +98,16 @@ Future<String?> getSavePath({
       confirmButtonText: confirmButtonText);
 }
 
-/// Gets a directory path from a user's file system
+/// Opens a directory selection dialog and returns the path chosen by the user.
```

Comment:
```suggestion
/// Opens a directory selection dialog and returns the path chosen by the user.
/// This always returns `null` on the web.
```

Reply:
I forget why we did that instead of making it throw unimplemented...